birdhouse/components/README.rst

```diff
@@ -56,12 +56,12 @@ Given the unattended nature, there is no UI.  Logs are used to keep trace.
 How to Enable the Component
 ---------------------------
 
-- Edit ``env.local`` (a copy of env.local.example_)
+- Edit ``env.local`` (a copy of `env.local.example`_)
```

Comment:
Not sure what's the difference surrounding `env.local.example` with backticks changes the rendering display.  Here's the original version from `master` https://github.com/bird-house/birdhouse-deploy/tree/e10a2ef9bd77498f1af22267cad5d181d0da65ef/birdhouse/components I see no difference.

Reply:
There's no difference technically, but parsers detect it more easily for rendering in IDEs.